src/main/java/by/exadel/internship/entity/Form.java

```diff
@@ -77,7 +81,7 @@
     private FormStatus formStatus;
 
     @OneToOne(cascade = CascadeType.ALL)
```

Comment:
I think the internship should not depend on the form

Reply:
done